src/quo/components/tabs/tabs/view.cljs

```diff
@@ -180,6 +177,15 @@
                                   :fade-end?           fade-end?
                                   :fading              fading
                                   :on-scroll           on-scroll})
+             :on-layout (fn []
```

Comment:
please use `rn/use-callback` hook

Reply:
for that, you need to refactor the entire component, and remove reagent render, feel free to ping me in Discord if you need help